engine/src/main/java/nl/inl/blacklab/search/results/Hits.java

```diff
@@ -408,8 +430,9 @@ public static Hits fromSpanQuery(QueryInfo queryInfo, BLSpanQuery query, SearchS
      * @param end hit ends
      * @return hits found
      */
-    public static Hits fromArrays(QueryInfo queryInfo, int[] docs, int[] starts, int[] ends) {
-        return new HitsList(queryInfo, new HitsArrays(new IntArrayList(docs), new IntArrayList(starts), new IntArrayList(ends)), null);
+    public static Hits fromArrays(QueryInfo queryInfo, int[] docs, int[] starts, int[] ends ) {
```

Comment:
This is only being used from testing code right now, so it is safe for now, we should expand on this to receive a indices array as well